pkg/controlplane/instance.go

```diff
@@ -660,7 +658,6 @@ func DefaultAPIResourceConfigSource() *serverstorage.ResourceConfig {
 		eventsv1beta1.SchemeGroupVersion,
 		networkingapiv1.SchemeGroupVersion,
 		nodev1.SchemeGroupVersion,
-		nodev1beta1.SchemeGroupVersion,
```

Comment:
v1beta1 should be left here until 1.26. RuntimeClass will automatically default to not being served in 1.25, but can be force re-enabled for one release with `KUBE_APISERVER_SERVE_REMOVED_APIS_FOR_ONE_RELEASE=true`

Reply:
this comment is still outstanding